glide.yaml

```diff
@@ -0,0 +1,22 @@
+package: github.com/skatteetaten/radish
+import:
+- package: github.com/drone/envsubst
+  version: ^1.0.2
+- package: github.com/kballard/go-shellquote
+- package: github.com/magiconair/properties
+  version: ^1.8.1
+- package: github.com/pkg/errors
+  version: ^0.8.1
+- package: github.com/plaid/go-envvar
```

Comment:
Har du behov for å dra inn lib for å håndtere env vars ? Er det ikke nok å bruke det som tilbys av golang std